content/tutorials/development_docker/index.qmd

```diff
@@ -260,12 +262,13 @@ This is a simple and quick way to run R and RStudio in a container.
 
 However, there are limitations:
 
-::: {.callout-warning}
+::: {.callout-note}
 - You have to live with the R packages provided in the container, or otherwise install them each time you access it...
 - ... unless you make your container permanent by omitting the `--rm` option. Note that this will cost considerable disk space, will not transfer to other computers (the original purpose of Docker), and demand occasional updates ([@sec-permanence]). 
 - You could alternatively add `--pull always` to `docker run`, which will check and pull new versions.
 - Speaking of updates: it is good practice to keep software up to date. Occasionally update or simply re-install your Docker image and R packages to get the latest versions.
 - You should make sure that the containers are configured correctly and securely. This is especially important with server components which expose your machine to the internet.
+- Because most containers contain a linux system, user permissions are taken seriously, and the consequences might be confusing. There are guides online ([e.g. here](https://labex.io/tutorials/docker-how-to-handle-permissions-in-docker-415866)); there are example repositories (like the author's own struggle [here](https://github.com/inbo/containbo?tab=readme-ov-file#understanding-volumes) and [here](https://github.com/inbo/containbo/tree/main/emacs)); base images are well set up and one can normally get by with default users.
```

Comment:
```suggestion
- Because most containers contain a Linux system, user permissions are taken seriously, and the consequences might be confusing. There are guides online ([e.g. here](https://labex.io/tutorials/docker-how-to-handle-permissions-in-docker-415866)); there are example repositories (like the author's own struggle [here](https://github.com/inbo/containbo?tab=readme-ov-file#understanding-volumes) and [here](https://github.com/inbo/containbo/tree/main/emacs)); base images are well set up and one can normally get by with default users.
```